src/index.js

```diff
@@ -0,0 +1,90 @@
+import './style.css';
+import Tasks from './task.js';
+import MenuIcon from './icons/menu-vertical.png';
+import taskUpdate from './taskUpdate.js';
+
+const clear = document.querySelector('.clear');
+const addTask = document.getElementById('add-task');
+
+const task = new Tasks();
+let id = 0;
+
+const removeTask = (event) => {
+  const id = event.target.id.split('-');
+  const taskItem = document.getElementById(`${id[1]}`);
+  const checkbox = document.getElementById(`checkbox-${id[1]}`);
+  taskItem.classList.toggle('line');
+  checkbox.classList.toggle('completed');
+};
+
+const changeTask = (taskInput) => {
+  taskInput.addEventListener('change', (event) => {
+    task.taskList = Tasks.fetch();
+    task.taskList.forEach((item) => {
+      if (item.index === parseInt(event.target.id, 10)) {
+        item.description = event.target.value;
+      }
+      task.updateIndex();
+      Tasks.updateStorage(task.taskList);
+    });
+  });
+};
+
+const activeTask = (taskInput) => {
+  taskInput.addEventListener('click', () => {
+    const current = document.getElementsByClassName('active');
+    if (current.length > 0) {
+      current[0].className = current[0].className.replace('active', '');
+      current[0].className = current[0].className.replace('active', '');
+    }
+    taskInput.classList.add('active');
+    const listNode = taskInput.parentNode.parentNode;
+    listNode.classList.add('active');
+  });
+};
+
+const createTaskRow = (id, desc) => {
+  const allList = `
+      <li class="task-list" id="task-list-${id}">
+      <span><button class="checkbox" alt="checkbox" name="checkbox" id="checkbox-${id}"></button></span>
+      <span class="task-name"><input type="text" class="add-task task-item" id="${id}" value="${desc}"></span>
+      <span><img src="${MenuIcon}" alt="Vertical Menu Icon"/></span>
+    </li>`;
+  clear.insertAdjacentHTML('beforebegin', allList);
+
+  document.querySelectorAll('.checkbox').forEach((elem) => {
+    elem.addEventListener('click', removeTask);
+    elem.addEventListener('click', taskUpdate);
+  });
+  const taskInput = document.getElementById(`${id}`);
+  changeTask(taskInput);
+  activeTask(taskInput);
+};
+
+task.taskList = Tasks.fetch();
+if (task.taskList) {
+  task.taskList.forEach((task) => {
+    createTaskRow(task.index, task.description);
+  });
+}
+
+addTask.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') {
```

Comment:
Here you listened for a key press on "Enter", I was hoping you'd consider listening for a click on the icon inside of the input box instead as that will ensure a more friendly and direct user experience approach than a keypress.